### ISSUE TITLE
TrackEval

### ISSUE THREAD

**Issue by johnnynunez** [mcv-m6-video/class2023#15]
Does TrackEval evaluate really MTMC? I think that it's only for single camera.
If we changed id not affects in results, but yes in qualitative results

**Comment by jrhupc** [comment]
@johnnynunez I am not sure if I understand, the multi-camera evaluation is the same as the single-camera, just that the groundtruth also provides the same ID for the tracking of the same object in different cameras. If you're able to do re-identification between cameras, you can assign same IDs to different cameras and join them together as a single track. So if re-arrange correctly metric evaluation can be done for multi-camera as in single-camera. 

In any case, this is another repository you can use for your metrics: https://github.com/cheind/py-motmetrics. Maybe there the multi-camera evaluation is clearer?

Best regards,

**Comment by johnnynunez** [comment]
> @johnnynunez I am not sure if I understand, the multi-camera evaluation is the same as the single-camera, just that the groundtruth also provides the same ID for the tracking of the same object in different cameras. If you're able to do re-identification between cameras, you can assign same IDs to different cameras and join them together as a single track. So if re-arrange correctly metric evaluation can be done for multi-camera as in single-camera.
> 
> In any case, this is another repository you can use for your metrics: https://github.com/cheind/py-motmetrics. Maybe there the multi-camera evaluation is clearer?
> 
> Best regards,

Exactly, but we have the same results that single camera, so it does not make sense

**Comment by jrhupc** [comment]
@johnnynunez , can you try the new repository to see if it helps?

**Comment by rmorros** [comment]
You can take a look at: https://github.com/JonathonLuiten/TrackEval/issues/31
